packages/react-native-gesture-handler/src/v3/HostGestureDetector.web.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useRef } from 'react';
+import { View } from 'react-native';
+import RNGestureHandlerModuleWeb from '../RNGestureHandlerModule.web';
+import { ActionType } from '../ActionType';
+export interface GestureHandlerDetectorProps {
+  handlerTags: number[];
+  dispatchesAnimatedEvents: boolean;
+  moduleId: number;
+  children?: React.ReactNode;
+}
+
+const HostGestureDetector = (props: GestureHandlerDetectorProps) => {
+  const { handlerTags, children } = props;
+
+  const viewRef = useRef(null);
+  const propsRef = useRef<GestureHandlerDetectorProps>(props);
+  const oldHandlerTags = useRef<Set<number>>(new Set<number>());
+
+  const attachHandlers = (): void => {
```

Comment:
You don't have to specify that it returns `void`. I don't say that it is bad, but `HostGestureDetector` is an arrow function too and it doesn't have return type specified by hand. So let's stick to one convention.